functions/New-DbaAvailabilityGroup.ps1

```diff
@@ -12,7 +12,7 @@ function New-DbaAvailabilityGroup {
         * Adds secondary replica if supplied
         * Adds databases if supplied
             * Performs backup/restore if seeding mode is manual
-            * Performs backup to NUL if seeding mode is automatic
```

Comment:
I think this is still good information for people to know (maybe it can impact their lsn chain?). I like your addition, but let's keep both 👍🏼